addon/serializers/application.js

```diff
@@ -190,8 +190,8 @@ export default DS.RESTSerializer.extend({
       hasMany.forEach( function( child ) {
         json[key].objects.push({
           '__type'    : 'Pointer',
-          'className' : child.parseClassName(),
-          'objectId'  : child.get( 'id' )
+          'className' : _this.parseClassName(child.type.typeKey),
```

Comment:
The `serializeBelongsTo` in this code does not appear to be the version @Kira2 was talking about.